aws/aws-event-streams/src/main/java/software/amazon/smithy/java/aws/events/RpcEventStreamsRequest.java

```diff
@@ -0,0 +1,42 @@
+/*
+ * Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package software.amazon.smithy.java.aws.events;
+
+import java.nio.ByteBuffer;
+import java.util.concurrent.Flow;
+import software.amazon.smithy.java.core.schema.Schema;
+import software.amazon.smithy.java.core.schema.SerializableStruct;
+import software.amazon.smithy.java.core.schema.TraitKey;
+import software.amazon.smithy.java.core.serde.event.EventEncoderFactory;
+import software.amazon.smithy.java.core.serde.event.EventStreamFrameEncodingProcessor;
+
+/**
+ * A util class to serialize event streams requests for RPC protocols.
+ */
+public final class RpcEventStreamsRequest {
```

Comment:
The name threw me off. Maybe RpcEventStreamsUtils would be more clear that this isn't actually a request, but request utils.

Reply:
Sure.